doc/tutorial/text_analytics/working_with_text_data.rst

```diff
@@ -370,19 +370,21 @@ analysis of the results::
   ...                                         # doctest: +NORMALIZE_WHITESPACE
                           precision    recall  f1-score   support
   <BLANKLINE>
-             alt.atheism       0.95      0.81      0.87       319
-           comp.graphics       0.88      0.97      0.92       389
-                 sci.med       0.94      0.90      0.92       396
-  soc.religion.christian       0.90      0.95      0.93       398
+             alt.atheism       0.97      0.60      0.74       319
```

Comment:
why those scores changed?

Reply:
+1. I don't understand.

Reply:
I agree this is strange. Running the doctests locally I can confirm the former values are the correct ones.